addOns/webdrivers/webdrivers.gradle.kts

```diff
@@ -4,8 +4,8 @@ import org.zaproxy.gradle.tasks.DownloadWebDriver
 
 description = "Common configuration of the WebDriver add-ons."
 
-val geckodriverVersion = "0.23.0"
-val chromeDriverVersion = "2.43"
+val geckodriverVersion = "0.24.0"
+val chromeDriverVersion = "74.0.3729.6"
```

Comment:
This should be updated to 73, which is the current stable release.

Reply:
@thc202, thank you for pointing that. fixed it.